app/assets/javascripts/discourse/app/lib/text.js

```diff
@@ -1,44 +1,17 @@
-import PrettyText, { buildOptions } from "pretty-text/pretty-text";
```

Comment:
Most of the actual implementations are now in `app/static/markdown-it/index.js` so that the code can be loaded on-demand. These are now just stubs that loads that bundle and then call the synchronous methods defined there.